src/cli/cli.cpp

```diff
@@ -560,8 +560,7 @@ void Interpreter::ProcessChannel(int argc, char *argv[])
                 mServer->OutputFormat("count: %lu\r\n", otChannelMonitorGetSampleCount(mInstance));
 
                 mServer->OutputFormat("occupancies:\r\n");
-                for (uint8_t channel = otLinkGetPhyChannelMin(mInstance); channel <= otLinkGetPhyChannelMax(mInstance);
-                     channel++)
+                for (uint8_t channel = Phy::kChannelMin; channel <= Phy::kChannelMax; channel++)
```

Comment:
`Phy::kChannelMin` and `Phy::kChannelMax` are not defined in public header files.

Reply:
Ok, I see. So `otLinkGetPhyChannelMin()` and `otLinkGetPhyChannelMax()` APIs are still required here. Reverted.

Reply:
Is it possible to use the channel mask directly?

Reply:
I saw some behavior changes between using ChannelMask VS ChannelMin/Max. If we use channelmask, if will start from 0 every time, and only handle the supported channels.

If it is acceptable, can I submit a separate PR to remove the `otLinkGetPhyChannelMin()` and `otLinkGetPhyChannelMax()` related usage? So to keep this PR only focus on adding preferred channelmask feature.